sys-apps/ignition/files/0005-config-add-ignition-translation.patch

```diff
@@ -176,12 +176,8 @@ index 00000000..347d148c
 +
 +	if _, ok := fsMap[name]; !ok {
 +		addedSuffixCounter += 1
-+		if name == "oem" {
-+			fsMap[name] = "/usr/share/oem"
-+		} else {
-+			// generate a new path
-+			fsMap[name] = "/tmp/" + name + "-ign" + strconv.FormatUint(addedSuffixCounter, 10)
```

Comment:
I don't understand why this is deleted (for the OEM filesystem we want to use the existing mount point)

Reply:
Ah, I read the linked comment in the PR description… not really sure if the result would still be correct.
Reminds me that we have to move the `/usr/share/oem` mount point anyway to be `/oem` or similar because of systemd-sysext dropping it when run manually.

Reply:
This events of oem mount handling in the initrd are a bit messy currently, maybe your patch would even help a bit with the downside that it would have two mount points but I think that's fine!
I've tested it and found some unlabeled files/dirs but I think that's ok for now.

Reply:
Normally, what kind of files are in `/usr/share/oem`? Probably that's not something you'd normally find in either the selinux policies coming from the selinux project nor the fedora one. so you might need a custom policy depending on the files that are there.

Reply:
Ignition creates the `/etc/systemd/` folder and the preset file `/etc/systemd/system-preset/20-ignition.preset` unlabeled.

Reply:
Maybe because policies are not loaded in the initrd?

Reply:
With selinux not being loaded I would assume that every action in the initrd does not result in correct labels, or? And even if it would be loaded, the path `/sysroot/etc/…` is different from `/etc` and the labeling wouldn't work either. Maybe the setfiles step should be changed to cover the directories created on the way from `MkdirForFile`?
Anyway, to me it looks like `setfiles` being  a no op in this case because it gets called before the file gets created by `s.EnableUnit(unitString)`/`s.DisableUnit(unitString)`.

Reply:
Ah, no, the reabeling is done in the and and uses the first missing part, then it should be ok.

Reply:
Note: Using `/oem` as OEM mountpoint should work and it won't be a read-only path. Using /tmp/something is also ok.